policyengine_us/variables/household/demographic/person/partner_is_disabled.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class partner_is_disabled(Variable):
```

Comment:
There is already a person-level variable called `is_disabled` and a taxunit-level variable called `disabled_spouse`.  The current name is an attempt to make a strong distinction between it and those two.   So, no, I don't think the name change you suggest is a good idea.

Reply:
The two existing variables describe the disability status of the spouse.  The new variable specified whether or not you marriage partner is disabled.  It says nothing about your own disability status.

Reply:
OK how about `has_disabled_spouse` then? I understand that the variable name may be hard to disambiguate across entities, but I don't think introducing a synonym for `spouse` addresses this, and it would make our naming system less internally consistent.

Reply:
Done.